requirements-base.txt

```diff
@@ -38,6 +38,8 @@ maxminddb>=2.3
 mistune>=2.0.3
 mmh3>=4.0.0
 packaging>=24.1
+pandas>=2.2.3
+pandas-stubs>=2.2.3.250308
 parsimonious>=0.10.0
```

Comment:
per the discussion in slack -- adding these dependencies to sentry is a non-starter